<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EgaliSignage</class>
 <widget class="QMainWindow" name="EgaliSignage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EgaliSignage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="image_main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>650</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_guidance">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>700</width>
      <height>118</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_secondary">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>324</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
